packages/scss/src/components/_button.scss

```diff
@@ -398,91 +402,120 @@
 // WRAPPER
 // ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
+ul.button-group {
```

Comment:
Separating `ul.button-group` and `button-group` is for legacy support ?

Maybe we could juste merge the two with a `mod-` ? Or with something like
```
.button-group {
   &:is(ul){
      ...
   }
}
```
I think that `button-group` should be in is own .scss file